src/calendar/view/CalendarViewModel.ts

```diff
@@ -59,15 +59,16 @@ export class CalendarViewModel implements EventDragHandlerCallbacks {
 	 */
 	private previewedEvent: { event: CalendarEvent; model: CalendarEventPreviewViewModel | null } | null = null
 
-	_hiddenCalendars: Set<Id>
-	// Events that have been dropped but still need to be rendered as temporary while waiting for entity updates.
+	private _hiddenCalendars: Set<Id>
+	/** Events that have been dropped but still need to be rendered as temporary while waiting for entity updates. */
+	// visible for tests
```

Comment:
It is bad practice to test using private member variables. Tests should only care about whether the code returns what is expected of it based on the codes specification. Private member variables are solely an implementation detail.

For example, assert on `temporaryEvents()` instead of `_draggedEvent.eventClone` & `_transientEvents`.
Use:
```ts
o(viewModel.temporaryEvents).deepEquals([])
```
Instead of:
```ts
o(viewModel._draggedEvent?.eventClone).equals(undefined)
o(viewModel._transientEvents).deepEquals([])
```

Reply:
I am mostly with you on this but I have not introduced it nor gonna change it in this PR,  I just  wanted to document it

Reply:
I had another look at it and I confirm once again that in many cases it's not a simple mechanical change and to test it without resorting to grabbing private fields would either need a separate component or some other kind of refactoring that's out of scope. In general I would also say that it's testing the wrong thing if it's checking internal state of multiple fields (who cares about it on the outside?). If you insist on changing this I would suggest making a technical task and scheduling it.

Reply:
Thanks for taking a look into it. In that case, we can leave it for now. I agree with you completely about asserting on internal state. We could create a technical task to review whether our tests are appropriate for the level they are on.